flow2ml/Flow.py

```diff
@@ -202,3 +204,15 @@ def getDataset(self,img_dimensions,train_val_split):
     self.update_progress( 100/100.0,"Created Datasets" ) 
 
     return self.dataset
+
+  def deployTensorflowModels(self):
+    '''Deploy conversion from tensorflow model to tensorflowjs or tflite model'''
+
+    if(self.model['tfjs']==True):
+      tfjs.converters.save_keras_model(model, 'models')
+
+    elif self.model['tflite']==True:
+      TF_LITE_MODEL_FILE_NAME = "tf_lite_model.tflite"
+      tf_lite_converter = tf.lite.TFLiteConverter.from_keras_model(model)
```

Comment:
The use of the tensorflow function is wrong. 
*tf.lite.TFLiteConverter.from_keras_model()* takes in the model directory as an input.